proto/process/agent.proto

```diff
@@ -669,6 +755,7 @@ enum K8sResource {
     CR = 21;
     VERTICALPODAUTOSCALER = 22;
     HORIZONTALPODAUTOSCALER = 23;
+    ECSTASK = 24;
```

Comment:
that looks odd to add ECS resources to the `K8sResource` message, should probably have its own enum

Reply:
https://github.com/DataDog/agent-payload/blob/77eb97fe14d9d6ffa0b8758d09d192cdacf4b018/proto/process/agent.proto#L768